### ISSUE TITLE
Fontawesome 509 ms delay

### ISSUE THREAD

**Issue by inzerat** [buddycore/surface-theme#67]
It refers to an external source and there is a delay 509 ms which is enough. It would fontawesome Please refer internally to surface files? It will benefit all users.

![untitled2](https://cloud.githubusercontent.com/assets/21200726/20486618/b6b7a440-b000-11e6-88e7-1a9626af1421.png)

**Comment by inzerat** [comment]
This external source please downloading intern too 

```
fonts.gstatic.com
fonts.googleapis.com
```

**Comment by buddycore** [comment]
I have updated with local copy of Font Awesome, please tell me your results with latest version. Thanks

**Comment by inzerat** [comment]
Yes much better time. thanks. Maybe too this two external sources 
`fonts.gstatic.com
fonts.googleapis.com`
because google have sometimes crash issue and surface is slow
